chart/values.yaml

```diff
@@ -237,7 +234,7 @@ accountsWww:
   image:
     registry: gcr.io
     repository: carto-onprem-artifacts/accounts-www
-    tag: "2021.12.27.01"
+    tag: "2022.02.10"
```

Comment:
I think that, in general, we should implement to use the `.Chart.AppVersion` to avoid editing this too much.

Reply:
Yes, i agree.

Reply:
Take into account that using `.Chart.AppVersion` along all the images forces you to update a new tag in every container even when you only push changes on one of them, maybe it is ok for you but should be taken into account.